source/instructors/teaching/grading/nbgrader.rst

```diff
@@ -1,18 +1,24 @@
 .. meta::
    :description: Auto-Grade Jupyter notebook assignments using nbgrader.
    
+.. _notebooks:
+
 
 Auto-Grade with nbgrader
 ========================
 Codio supports `Jupyter notebook <https://jupyter.org/>`_ auto-grading functionality through `nbgrader <http://nbgrader.readthedocs.io/en/stable/index.html>`_. Assignments are created with `Jupyter notebook <https://jupyter.org/>`_ and when the assignment is published to a course, the release version is created for the student. If the assignment is updated and reublished, it overwrites all tests and read-only cells with the new version and the release version for the students is updated.
 
 When a student submits the assignment by marking the assignment as complete, the assignment is automatically graded. However, manual grading is also possible if desired. 
 
-.. Note:: Any user configurations for nbgrader should be stored in a **.codio-jupyter** file. If a **.codio-jupyter** file is used in a project, Codio assumes it is the Jupyter based grader so only **nbgrader** can be selected for the assessment scripts in the assignment.
+User configurations for nbgrader can be stored in a **nbgrader_config.py** or in **.codio-jupyter** file. A **.codio-jupyter** file is requited in a project in order that Codio assumes it is the Jupyter based grader so only **nbgrader** can be selected for the assessment scripts in the assignment.  
```

Comment:
A **.codio-jupyter** file must be present in a project to let Codio know that **nbgrader** should be used to grade Jupyter assessments.